cdap-ui/app/features/hydratorplusplus/templates/partial/node-config-modal/popover-template.html

```diff
@@ -0,0 +1,26 @@
+<!--
+  Copyright © 2016 Cask Data, Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+-->
+<div class="modal"
+      ng-class="{in: animate}"
+      ng-style="{'z-index': 1050 + index*10, display: 'block'}"
+      data-ng-click="close($event)">
+  <div class="modal-dialog modal-lg"
+       ng-class="{'expand': $parent.HydratorPlusPlusNodeConfigCtrl.state.windowMode == 'expand'}">
```

Comment:
I am not entirely sure if we need type to here. Its just sting check and I am not sure if we need to check type equality here. Its generally preferred to use `===` but it comes with drawback of one additional check (type check) which I am not sure if we need here.